packages/server/package.json

```diff
@@ -77,6 +77,7 @@
     "graphql": "0.13.0",
     "graphql-auth": "0.2.2",
     "graphql-iso-date": "^3.3.0",
+    "graphql-playground-middleware-express": "^1.7.1",
```

Comment:
Why are you adding this package in this PR, it is not related to this PR. If you want to add support for GraphQL playground IDE, it is okay, but it should be done in separate Pull Request

Reply:
Ok